src/client/rest/RESTMethods.js

```diff
@@ -60,6 +60,12 @@ class RESTMethods {
     return new Promise((resolve, reject) => { // eslint-disable-line complexity
       if (typeof content !== 'undefined') content = this.client.resolver.resolveString(content);
 
+      // nonce has to be a uint64 :<
+      if (nonce !== undefined) {
+        nonce = Math.abs(parseInt(nonce));
```

Comment:
Why? Just add to the validation to ensure it's positive - don't coerce negatives into positives.